leda/components/Button/theme.ts

```diff
@@ -1,7 +1,9 @@
 /* eslint-disable key-spacing */
 export const defaultButtonTheme = {
+  // todo replace classes when base theme is ready
   disabled:                 'disabled',
   loading:                  'loading',
   wrapper:                  'button-wrapper',
+  // wrapper:                  'ld__btn',
```

Comment:
Not needed?

Reply:
Is ld like a prefix?

Reply:
Btw BEM naming will make you regret eventually, trust me)

Reply:
is needed, there is a todo, new classes will be used as soon as all components are ready, base will replace leda.light

ld is rather a namespace, no BEM intended =^..^=

we have main classes and modificators, main classes will be namespaced/prefixed, but tbd if any concerns